python_modules/libraries/dagster-dbt/dagster_dbt/utils.py

```diff
@@ -44,7 +44,7 @@ def input_name_fn(dbt_resource_props: Mapping[str, Any]) -> str:
 
 
 def output_name_fn(dbt_resource_props: Mapping[str, Any]) -> str:
-    return dbt_resource_props["unique_id"].split(".")[-1]
+    return dbt_resource_props["unique_id"].split(".")[-1].replace("-", "_")
```

Comment:
Adding a comment on why we are doing this transformation would be nice

Reply:
sounds good